samples/mpu/mem_domain_apis_test/src/main.c

```diff
@@ -76,7 +93,17 @@ void app_thread(void *dummy1, void *dummy2, void *dummy3)
 
 	while (1) {
 		printk("app thread %d (%p) yields.\n", id, &app_thread_id[id]);
-
+#ifdef CONFIG_x86		
```

Comment:
get rid of this.
these tests need to be arch-agnostic and verify the public APIs.
shouldn't be making arch calls here

Reply:
This was during debugging. Will remove this.